docs/eth_hash.backends.rst

```diff
@@ -0,0 +1,30 @@
+eth\_hash\.backends package
+===========================
+
+Submodules
+----------
+
+eth\_hash\.backends\.pycryptodome module
+----------------------------------------
+
+.. automodule:: eth_hash.backends.pycryptodome
+    :members:
+    :undoc-members:
+    :show-inheritance:
+
+eth\_hash\.backends\.pysha3 module
+----------------------------------
+
+.. automodule:: eth_hash.backends.pysha3
+    :members:
```

Comment:
These look like generated files that should be git ignored??

Reply:
Kind of, except, they're really meant to be to be starting points that you go in and edit. For example, I deleted some headers out of https://github.com/ethereum/eth-hash/pull/2/files/105f9c08d422f7c994a23179779bdb2414cbc297#diff-443171bd924359fed75ad812c3ad20b2

Also, sphinx autodoc by default won't replace the file during generation if it's already there. So checking in the templates seems like the best way to consistently generate the same results in everyone's local doc builds.